dist/ghp-updater.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
This will expose the token in the logs due to the `set -x` earlier. We should probably remove the `-x` from it:

```suggestion
set -e
```